members-list.md

```diff
@@ -18,6 +18,11 @@ Also don't be a cop-out by saying something like "I am happy/looking forward to
 ## The list 
 [Add your details here]`<--Don't remove this line`<br/>
 
+**Name**:  Aayudh Bhattacharya  <br/>
+**SRN**:  PES2UG20CS006
```

Comment:
Hi! There's a `<br/>` missing at the end of this line. It looks a bit weird on the file.

![image](https://user-images.githubusercontent.com/34161949/104891165-93870d00-5996-11eb-99f0-9ffe695dc1bb.png)

To fix that, either add two spaces at the end of this line (line 22) or add a `<br/>`.